src/main/java/rx/internal/schedulers/NewThreadWorker.java

```diff
@@ -50,6 +53,15 @@
         PURGE = new AtomicReference<ScheduledExecutorService>();
         PURGE_FORCE = Boolean.getBoolean(PURGE_FORCE_KEY);
         PURGE_FREQUENCY = Integer.getInteger(FREQUENCY_KEY, 1000);
+
+        final Integer androidApiVersion = PlatformDependent.getAndroidApiVersion();
+
+        // Cached to prevent method call
+        IS_ANDROID = androidApiVersion != null;
```

Comment:
You can still eliminate this variable altogether. The same check is included in the other.